README.md

```diff
@@ -86,36 +99,18 @@ If you change the tree-sitter grammar source file, `cargo watch` doesn't seem to
 
 5. Run unit tests
 
-```
-cargo test
+```sh
+cargo build && cargo test
 ```
 
 6. Run integration tests
 
-```
-cargo build && EXO_RUN_INTROSPECTION_TESTS=true target/debug/exo test integration-tests
+```sh
+cargo build && EXO_RUN_INTROSPECTION_TESTS=true cargo run --bin exo test integration-tests
 ```
 
 ## Logging, telemetry and tracing
 
 The code is instrumented using the [tracing](https://crates.io/crates/tracing) framework and will output log events to the console by default. The log level can be configured by setting the `EXO_LOG` variable which behaves identically to `RUST_LOG`. It defaults to `info` but can be set to other standard log levels such as `debug` (which will also show logging from libraries such as `tokio-postgres`). More [sophisticated settings](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/struct.EnvFilter.html) can also be used to tune the output for specific crates and modules.
 
```

Comment:
Maybe we can also drop the sentence beginning "More sophisticated settings" since we have an example in the external docs (we could maybe add a link to `EnvFilter` there). The current link is also wrong. It should be https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html

Reply:
Fixed. Move some part to the main docs.